src/table/pagination-table/components/head/HeadCell.tsx

```diff
@@ -70,6 +73,9 @@ const HeadCell = ({ column, columnIndex, columnsLength }: HeadCellProps) => {
 
   const handleOpenMenu = () => setOpen(true);
 
+  const newHeadCellRelatedLogic = isNewHeadCellMenuEnabled ? column.colIdx === 0 : false;
+  const tabIndex = newHeadCellRelatedLogic && isInteractionEnabled && areTabStopsEnabled(keyboard) ? 0 : -1;
```

Comment:
Yes, why not 🙂 
when flag is on -> tab stop should be for entire cells 
when it's off -> it will be the responsibility of `StyledSortButton` and ... menu (within <HeadCellContent /> component) to handle tab stops

Reply:
hmm no idea what I thought here, you are right. of course when the flag is of, no focus on the cell itself